src/App.js

```diff
@@ -17,10 +18,11 @@ function App() {
 
     return (
         <AppContext.Provider className="App" value={[context, setContext]}>
+          <div className="pageWrap">
```

Comment:
Add this className to the AppContext instead, no need for a separate div.

Reply:
Also, prefer one style of naming for CSS classes (not sure which). Either camelCase or kebab-case would be my suggestions.

Reply:
kebab-case would be good to be agreed upon.

Reply:
YES